src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -1295,15 +1306,12 @@ To delete a task from the workflow, the user should run the pragma `#%delete_tas
 
 ==== Create a job
 
-To create a job, use the `#%job()` pragma:
+To create a job, specify job variables and/or job generic information, use the `#%job()` pragma:
```

Comment:
`specify job variables and/or job generic information by using the `#%job()` pragma:` ?

Reply:
Not with a sentence starting by "To create" ;)